Gemfile

```
@@ -62,7 +62,7 @@ gem "pg",                                                    :require => false
 gem "pg-dsn_parser",                    "~>0.1.0",           :require => false
 gem "psych",                            "~>3.1",             :require => false # This can be dropped once we drop ruby 2.5
 gem "query_relation",                   "~>0.1.0",           :require => false
-gem "rails",                            "~>6.0.0"
+gem "rails",                            "~>6.0.0", ">=6.0.3.7"
```

Since the minimum z stream is bump too, can you change this to 

```suggestion
gem "rails",                            "~>6.0.3", ">=6.0.3.7"
```

---

👍